translations/es/LC_MESSAGES/messages.po

```diff
@@ -3,108 +3,168 @@
+"{invalid_argument} a {allowed_types}."
 
 msgid "Invalid Argument Type"
-msgstr "No puedes usar {command} con {invalid_argument} porque es {invalid_type}. Intenta cambiar {invalid_argument} a {allowed_types}."
+msgstr ""
+"No puedes usar {command} con {invalid_argument} porque es {invalid_type}. "
+"Intenta cambiar {invalid_argument} a {allowed_types}."
 
 msgid "Invalid At Command"
-msgstr "El comando {at} no puede utilizarse a partir del nivel 16. Puedes usar corchetes para usar un elemento de una lista, por ejemplo `amigos[i]`, `números_de_suerte[{random}]`."
+msgstr ""
+"El comando {at} no puede utilizarse a partir del nivel 16. Puedes usar "
+"corchetes para usar un elemento de una lista, por ejemplo `amigos[i]`, "
+"`números_de_suerte[{random}]`."
 
 msgid "Invalid Space"
-msgstr "¡Oops! Has comenzado una línea con un espacio en la línea {line_number}. Los espacios confunden a los ordenadores, ¿podrías quitarlos?"
+msgstr ""
+"¡Oops! Has comenzado una línea con un espacio en la línea {line_number}. Los "
+"espacios confunden a los ordenadores, ¿podrías quitarlos?"
 
 msgid "Invalid Type Combination"
-msgstr "No puedes usar {invalid_argument} y {invalid_argument_2} con {command} porque uno es {invalid_type} y el otro es {invalid_type_2}. Intenta cambiar {invalid_argument} a {invalid_type_2} o {invalid_argument_2} a {invalid_type}."
+msgstr ""
+"No puedes usar {invalid_argument} y {invalid_argument_2} con {command} "
+"porque uno es {invalid_type} y el otro es {invalid_type_2}. Intenta cambiar "
+"{invalid_argument} a {invalid_type_2} o {invalid_argument_2} a "
+"{invalid_type}."
 
 msgid "Locked Language Feature"
-msgstr "¡Estás usando {concept}! Es genial, pero ¡{concept} todavía no está desbloqueado! Se desbloqueará en un nivel posterior."
+msgstr ""
+"¡Estás usando {concept}! Es genial, pero ¡{concept} todavía no está "
+"desbloqueado! Se desbloqueará en un nivel posterior."
 
 msgid "Lonely Echo"
-msgstr "Has utilizado un {echo} antes de un {ask}, o un {echo} sin un {ask}. Coloca un {ask} antes del {echo}."
+msgstr ""
+"Has utilizado un {echo} antes de un {ask}, o un {echo} sin un {ask}. Coloca "
+"un {ask} antes del {echo}."
 
 msgid "Lonely Text"
-msgstr "Parece que olvidaste usar un comando con el texto que pusiste en la línea {line_number}"
+msgstr ""
+"Parece que olvidaste usar un comando con el texto que pusiste en la línea "
+"{line_number}"
 
 msgid "Missing Command"
 msgstr "Parece que has olvidado de usar un comando en la línea {line_number}."
 
 msgid "Missing Inner Command"
-msgstr "Parece que has olvidad usar un comando en el {command} que has usado en la línea {line_number}."
+msgstr ""
+"Parece que has olvidad usar un comando en el {command} que has usado en la "
+"línea {line_number}."
 
 msgid "Misspelled At Command"
-msgstr ""
+msgstr "Parece que haz escrito incorrectamente el comando {command}, en su lugar escribiste {invalid_argument} en la línea {line_number}."
```

Comment:
Although, I just translated this one and it wasn't wrapped, so maybe we solved the problem, but Weblate only runs --no-wrap on labels that were edited?

Reply:
Someone that speaks a language other than Spanish, can please translate some text in other language to see what happens (@Mark-Giesen @Felienne)